docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -170,3 +168,16 @@ The following config options would recognize strings in this format (shown as a
 * `quarkus.http.limits.max-body-size`,
 * `quarkus.http.limits.max-header-size`
 ====
+
+== Servlet Config
```

Comment:
Should this section mention the required Quarkus Undertow dependency specifically?